### PR TITLE
Improvement: Update elixir tailwind example

### PR DESCRIPTION
**Pull request by tomxkay**
Build example components and integrate phoenix storybook.

Uses [Heroicons](https://heroicons.com/) instead of feathericons

Note: I generated a new phoenix app and layer the example components and storybook integration on top because some of the packages were old.

### CODE REVIEW

File: example/project-elixir-tailwind/lib/shipwright_elixir_tailwind_web/components/atoms/input.ex [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            ]}
+          >
+            <%= @hintText %>
+          </p>
+        <% end %>
+      </div>
+
+      <div
+        class={[
+          "flex flex-row relative border focus-within:border-button-active focus-within:w-full",
```
Comment (spreston4) [inline example/project-elixir-tailwind/lib/shipwright_elixir_tailwind_web/components/atoms/input.ex:63]:
```suggestion
          "flex flex-row relative border rounded focus-within:border-button-active focus-within:w-full",
```

File: example/project-elixir-tailwind/lib/shipwright_elixir_tailwind_web/components/organisms/card.ex [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      <% end %>
+
+      <div class="mb-[16px]">
+        <h6 class="h6 h6-400 pb-[4px]"><%= @title %></h6>
+        <p class="paragraph-2 text-type-black-secondary"><%= @text %></p>
+      </div>
+
+      <div class="w-full h-[1px] mb-[16px] bg-background-white-line"></div>
+      <div class="flex flex-row justify-end">
+        <Components.Atoms.Button.button variant="fill" label="Button" />
```
Comment (spreston4) [inline example/project-elixir-tailwind/lib/shipwright_elixir_tailwind_web/components/organisms/card.ex:30]: User should be able to customize the button text / variants

Comment (tomxkay) [reply example/project-elixir-tailwind/lib/shipwright_elixir_tailwind_web/components/organisms/card.ex]: I added the attributes that'll allow users to define them. I tried briefing into make it a dropdown in storybook i.e [fill, outline, ghost] but I haven't found a way to do that yet so you'll just have to type in the string to toggle between the variants.

**Review by spreston4 — APPROVED** [review]
LGTM

**Pull request merged**